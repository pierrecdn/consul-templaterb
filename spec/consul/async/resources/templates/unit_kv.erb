KV: <%= YAML.dump(kv('yaml/test.yml').get_value_yaml()) %>