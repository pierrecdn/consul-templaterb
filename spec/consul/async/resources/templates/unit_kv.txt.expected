KV: ---
this:
  is:
    only:
      a:
        test: true
